README.md

```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Install
+* Add `rg_instructor_analytics` to installed app 
+* Add to lms url: 
```

Comment:
Please add the path to the URL file

Reply:
It is commonly known information for a specialist, who install such things.

Reply:
You write readme for all potential users of the application not only DevOps engineers. So please add the path.